0.8/docs/devguide/utility-functions.md

```diff
@@ -20,8 +20,8 @@ The {{site.project_title}} `Base` prototype provides a set of useful convenience
 
 *   `toggleAttribute(name, bool, [node])`. Like `toggleClass`, but toggles the named boolean attribute.
 
-*   `attributeFollows(name, neo, old)`. Moves a boolean attribute from `old` to
-    `neo`, unsetting the attribute (if set) on `old` and setting it on `neo`.
+*   `attributeFollows(name, new, old)`. Moves a boolean attribute from `old` to
```

Comment:
`new` is a reserved word. I suspect that's why it's not like that already. How about "newVal" and "oldVal"?